.github/workflows/zizmor.yml

```diff
@@ -0,0 +1,33 @@
+name: GitHub Actions Security Analysis with zizmor 🌈
```

Comment:
Fine with changing this name, though I do wanna keep the rainbow in at least the step. ✨